datalad/interface/run.py

```diff
@@ -559,6 +569,10 @@ def run_command(cmd, dataset=None, inputs=None, outputs=None, expand=None,
         preparation and command execution. In this mode, the caller is
         responsible for ensuring that the state of the working tree is
         appropriate for recording the command's results.
+    saver : callable, optional
+        Must take a dataset instance, a list of paths to save, and a
+        message string as arguments and must record any changes done
+        to any content matching an entry in the path list.
```

Comment:
Current one returns a generator of results records... May be with documenting here on what is expected from saver to be returned?

Reply:
Added